cdap-ui/app/cdap/components/DatasetDetailedView/Tabs/AuditTab.js

```diff
@@ -32,12 +32,13 @@ export default function AuditTab({entity}) {
   url += `&sourceUrl=${encodedSource}`;
 
   return (
-    <div className="iframe-container">
+    <div className="embed-responsive embed-responsive-16by9">
```

Comment:
oh I didn't realize this is because of the scrolling issue. It should be fine i think